index.html

```diff
@@ -1,29 +1,233 @@
 <!DOCTYPE html>
 <html lang="en-ca">
   <head>
-    <!-- Set the page title -->
-    <title>Assignment 02</title>
-
-    <!-- Set the charset we're using to UTF-8 -->
+    <title>The Graveyard</title>
     <meta charset="UTF-8" />
-
-    <!-- Set the screens default zoom / scale to 1 -->
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    
-    <!-- Specifies the document mode for Internet Explorer -->
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-
-    <!-- We are using a CSS Reset file to standardize browser default css -->
     <link rel="stylesheet" href="reset.css" />
-
-    <!-- This is where all of our awesome CSS code goes -->
     <link rel="stylesheet" href="style.css" />
+    <link rel="icon shortcut" href="Website logo.png">
   </head>
+
+  <header>
+    <div class="header">
+      <div class="container--large">
+        <div class="header--bar">
+
+          <a href="index.html"><img class="header--bar--logo" src="Website logo.png" alt="Logo of game"></a>
+      
+            <h2>The Graveyard</h2>
+
+          <nav class="header--bar--nav">
+            <ul>
+              <li><a href="#">Home</a></li>
+              <li><a href="#">Players</a></li>
+              <li><a href="#">Rankings</a></li>
+              <li><a href="#">Vods</a></li>
+            </ul>
+          </nav>
+
+        </div>
+      </div>
+    </div>
+  </header>
   <body>
-    <!--
+    
+    <main>
+
```

Comment:
Oh no! 

The `<header>` should be inside the `<body>` tag! 

Please have a look at older exercises where we went through semantic HTML markup